easybuild/easyconfigs/s/scanpy/scanpy-1.7.2-foss-2020b.eb

```diff
@@ -0,0 +1,104 @@
+description = """Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
+ jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
+ and differential expression testing. The Python-based implementation efficiently deals with
+ datasets of more than one million cells.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'openmp': True}
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('Autotools', '20200321'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.7.1'),
+    ('CMake', '3.18.4'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('h5py', '3.1.0'),
+    ('matplotlib', '3.3.3'),
+    ('networkx', '2.5'),
+    ('numba', '0.52.0'),
+    ('pandas', '1.2.4'),
+    ('PyTables', '3.6.1'),
+    ('statsmodels', '0.12.1'),
+    ('scikit-learn', '0.23.2'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('tqdm', '4.60.0', {
+        'checksums': ['ebdebdb95e3477ceea267decfc0784859aa3df3e27e22d23b83e9b272bf157ae'],
+    }),
+    ('seaborn', '0.11.1', {
+        'checksums': ['44e78eaed937c5a87fc7a892c329a7cc091060b67ebd1d0d306b446a74ba01ad'],
+    }),
+    ('natsort', '7.1.1', {
+        'checksums': ['00c603a42365830c4722a2eb7663a25919551217ec09a243d3399fa8dd4ac403'],
+    }),
+    ('anndata', '0.7.6', {
+        'checksums': ['a3cc67bba9a4cd4b5984aec64c4f577c2d5a4695f4064027f8e6a9dac1f508b2'],
+    }),
+    ('six', '1.15.0', {
+        'checksums': ['30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259'],
+    }),
+    ('patsy', '0.5.1', {
+        'checksums': ['f115cec4201e1465cd58b9866b0b0e7b941caafec129869057405bfe5b5e3991'],
+    }),
+    ('pynndescent', '0.5.2', {
+        'checksums': ['d9fd22210b8d64368376ff392e876fb72fe3cda282396cfa6a59440ab6600771'],
+    }),
+    ('umap-learn', '0.3.10', {
+        'modulename': 'umap',
+        'checksums': ['21ce6b6d7486905318ba6458b5a9ba2cfb935878d30c24e6fba64ee3bd504d09'],
+    }),
+    ('get_version', '2.1', {
+        'use_pip': False,
+        'checksums': ['8156b526c2557537b8ca82241fa2b82b3da25939627398f6567dee31ba9725bc'],
+    }),
+    ('legacy-api-wrap', '1.2', {
+        'modulename': 'legacy_api_wrap',
+        'use_pip': False,
+        'checksums': ['034a44612da7e9943d3964363a98937ab54d55e3301075374abe0d521eb8101b'],
+    }),
+    ('stdlib-list', '0.8.0', {
+        'modulename': 'stdlib_list',
+        'checksums': ['a1e503719720d71e2ed70ed809b385c60cd3fb555ba7ec046b96360d30b16d9f'],
+    }),
+    ('sinfo', '0.3.1', {
+        'checksums': ['e1b2358808aded7b2ff00ea0cd4e6a2d978fb2a44ee9b15ac23d64a81bf62706'],
+    }),
+    ('texttable', '1.6.3', {
+        'checksums': ['ce0faf21aa77d806bbff22b107cc22cce68dc9438f97a2df32c93e9afa4ce436'],
+    }),
+    ('python-igraph', '0.9.1', {
+        'modulename': 'igraph',
+        'checksums': ['f1895a83c57180f03c50fb77334a4d5029327aefee14f03381e10d34a1794410'],
+    }),
+    ('leidenalg', '0.8.4', {
```

Comment:
For both `python-igraph` and `leidenalg` we have existing easyconfig files, it would be nice to update those and include these as proper dependencies instead?